python/lsst/cp/pipe/ptc.py

```diff
@@ -145,6 +145,11 @@ class MeasurePhotonTransferCurveTaskConfig(pexConfig.Config):
         doc="Sigma cut for outlier rejection in PTC.",
         default=5.0,
     )
+    sigmaCutAfwMathStatsControl = pexConfig.Field(
+        dtype=float,
+        doc="Sigma cut for afwMatch.StatisticsControl()",
```

Comment:
Typo I think: Match vs Math.

Reply:
I changed the name of the parameter to `nSigmaClipPtc`

Reply:
...but I did fix the typo in the docstring, thanks!